packages/office-ui-fabric-react/src/components/Link/__snapshots__/Link.test.tsx.snap

```diff
@@ -10,3 +11,25 @@ exports[`Link renders Link correctly 1`] = `
   I'm a link
 </a>
 `;
+
+exports[`Link renders Link with no href as a button 1`] = `
+<button
+  aria-disabled={undefined}
+  className="ms-Link"
+  onClick={[Function]}
+>
+  I'm a link as a button
+</button>
+`;
+
+exports[`Link renders disabled Link correctly 1`] = `
+<a
+  aria-disabled={true}
+  className="ms-Link is-disabled undefined"
```

Comment:
"undefined" on the classname seems like a bug

Reply:
@lynamemi can you double check that classname and styles.root are working? would be nice to not have this undefined value here.

Reply:
I followed up with you more on Teams for extra context, but essentially,  css("classname" + styles.foo) will always return "classname undefined" because of our jest config. There is a lot of precedence in fabric react for keeping it this way, but I can separate the two styles out on separate lines to get rid of undefined:
disabled && 'is-disabled
disabled && styles.disabled
- see David's comment below